dev_guide/templates.adoc

```diff
@@ -51,67 +56,153 @@ A template describes a set of resources intended to be used together that can be
 
 ====
 
-== Labels
-link:../architecture/kubernetes_model.html#label[Labels] are used to manage and organize generated resources, such as pods. The labels specified in the template are applied to every resource that is generated from the template. 
+== Creating a Configuration From a Template
+You can create a configuration from a template using the CLI or, if a template
+has been uploaded to your project or global template library, using the
+Management Console.
 
-There is also the ability to add labels in the template from the command line.
+Create a template JSON file, like the above example, then upload it with the
+CLI using the following process:
+
+=== Uploading a Template
+You can upload a template to your current project's template library by passing
+a JSON file with the following command:
 
 ****
-`$ osc process -f _<filename>_ -l name=otherLabel`
+`osc create -f _<filename>_`
```

Comment:
All throughout this topic, there are many example commands missing the `$` command prompt at the beginning.